src/server/contest/templates/config_error.html

```diff
@@ -0,0 +1,17 @@
+{% extends "header.html" %}
+
+{% block title %} No contest running {% endblock %}
+
+{% block body %}
+
+     <div class="p-desc">
+                <h2>
+                    <br>No contest running.
+                    <br>Go to the admin panel and create one.<br><br>
```

Comment:
We don't need this line and the `Create Config` link. It is not relevant to the users.